### PR TITLE
Update unredactor.tsv

### PR DESCRIPTION
**Pull request by Infinite-Zero**
added corrected input for Kyle Teel, (also updated line 1798, teating --> testing)

### CODE REVIEW

File: unredactor.tsv [hunk 1]
```diff
@@ -1795,7 +1795,7 @@ Aditya-Rohan-Singh	validation	Tantoo	This film for me and my wife is more entert
...
+Aditya-Rohan-Singh	testing	Hillerman	I hope Robert Redford continues to make more films like this. █████████'s books are wonderful, and as a young child raised in the Southwest his stories hit home! Adam Beach is a highly under rated and under used actor. Wake up Hollywood, not everyone thinks that your Mel Gibson's are cool! Many movie goer's today want to see films that make you think. I have seen all of the Redford/Hillerman series. They are thoughtful, scenic and have great plots. I'm hoping that if enough people write to Robert Redford he may decide to make a few more! Thank you Adam Beach and Tony Hillerman for great entertainment! If anyone get's a chance to read Tony Hillerman's latest book do so! It's great. I also recommend traveling through Arizona, New Mexico, Utah and Colorado. Stop at every view site and feel the setting of Hillerman's books. Amazing experience.
```
Comment (cegme) [inline unredactor.tsv:1798]: Excellent.

File: unredactor.tsv [hunk 2]
```diff
@@ -3332,3 +3332,93 @@ mitch3lljones	testing	Milton David Jr.	Also fantastic are the young actors Josep
...
+Infinite-Zero	training	Duck█Soup	Famous movies are subject to Freudian analysis: Possessed, The Matrix, The Birds, Psycho, Vertigo, █████████, Monkey Business, The Exorcist, The Testament of Dr Mabuse, Alien, Alien Resurrection, The Great Dictator, City Lights, The Tramp, Alice in Wonderland, The Wizard of Oz, Dr Strangelove, The Red Shoes, Fight Club, Dead of Night, The Conversation, Blue Velvet, Solaris, Stalker, Mulholland Drive, Lost Highway, Persona, In The Cut, Eyes Wide Shut, The Piano Teacher, Three Colours: Blue, Dogville, Frankenstein, The Ten Commandments, Saboteur, Rear Window, To Catch a Thief, North by Northwest, Star Wars, Dune, Kubanskie Kazaki, Ivan The Terrible, Pluto's Judgment Day (Walt Disney), Wild at Heart.<br /><br />You may wonder how the Marx Brothers come into play.
```
Comment (cegme) [inline unredactor.tsv:3353]: There is a weird extra redaction in the name here

**Review by cegme — CHANGES_REQUESTED** [review]
Do another quality control pass. It looks like you are merging multiple sentences but you are not leaving a space between the sentences.

**Comment by Infinite-Zero** [comment]
> Do another quality control pass. It looks like you are merging multiple sentences but you are not leaving a space between the sentences.

I am not sure what you mean with multiple sentences without space between. some of the examples were multiple sentences long, but I do not think that is strictly an issue. Could you clarify what you mean here

**Comment by cegme** [comment]
Make sure you pull the latest

**Comment by cegme** [comment]
Consider closing this pr and making a new one.

**Comment by Infinite-Zero** [comment]
> Consider closing this pr and making a new one.

I have been trying to make a new one, but I get this error (or others) when I try.

image

**Comment by cegme** [comment]
Can you use the GitHub interface? I need these conflicts resolved before I can merge. Do this in a new folder:

image

**Review by cegme — CHANGES_REQUESTED** [review]
Almost there do another quality check when pulling

**Comment by Infinite-Zero** [comment]
> Can you use the GitHub interface? I need these conflicts resolved before I can merge. Do this in a new folder:
> 
> image

I have tried this, but ran into the issue of not being able to push the file again.
image

**Comment by cegme** [comment]
You cannot push to my repository you have to push to your fork. Then you can do a pull request

**Pull request closed without merging**